queue/amqp.go

```diff
@@ -464,5 +478,24 @@ func fromDelivery(d *amqp.Delivery) (*Job, error) {
 	j.tag = d.DeliveryTag
 	j.raw = d.Body
 
+	if retries, ok := d.Headers[retriesHeader]; ok {
+		retries, ok := retries.(int32)
+		if !ok {
+			fmt.Printf("retrieving header retries error: %#v\n", ok)
```

Comment:
Leftover?

Reply:
Yes, it is. Removing it ...  :wastebasket: